keras/callbacks.py

```diff
@@ -1161,7 +1170,8 @@ def handle_value(k):
 
         if self.model.stop_training:
             # We set NA so that csv parsers do not fail for this last epoch.
-            logs = dict([(k, logs[k]) if k in logs else (k, 'NA') for k in self.keys])
+            logs = dict([(k, logs[k]) if k in logs else (k, 'NA')
+                         for k in self.keys])
```

Comment:
For readability, maybe we should unpack it on multiple lines? It's not really readable (not that it was the case before).